spec/models/multipart_guidance_spec.rb

```diff
@@ -82,9 +82,9 @@
 
   describe '#pages_for' do
     it 'filters the pages by audience' do
```

Comment:
If it's not using `audience:` anymore, can this example description be changed to reflect that? How about:

```suggestion
    it 'filters the pages by guidance_section' do
```